frontend/src/components/object/ObjectTable.vue

```diff
@@ -33,60 +44,54 @@ const { loading, objectList } = storeToRefs(useObjectStore());
     paginatorTemplate="RowsPerPageDropdown CurrentPageReport PrevPageLink NextPageLink "
     currentPageReportTemplate="{first}-{last} of {totalRecords}"
     :rowsPerPageOptions="[10, 20, 50]"
+    v-model:selection="selectedObjects"
   >
     <template #empty>
       <div v-if="!loading" class="flex justify-content-center">
         <h3>There are no objects associated with your account in this bucket.</h3>
       </div>
     </template>
-    <Column field="path" header="Path" />
-    <Column field="public" header="Public" />
-    <Column field="createdAt" header="Created">
+    <Column selectionMode="multiple" headerStyle="width: 3rem"></Column>
+    <Column field="path" header="Name" headerStyle="width: 25%" bodyClass="truncate">
       <template #body="{ data }">
-        {{ formatDateLong(data.createdAt) }}
+        <div v-if="data.name?.length > 150" v-tooltip.bottom="{ value: data.name }">
+          {{ data.name }}
+        </div>
+        <div v-else>
+          {{ data.name }}
+        </div>
       </template>
     </Column>
-    <Column field="updatedAt" header="Updated">
+    <Column field="id" header="Object ID" bodyClass="truncate">
+      <template #body="{ data }">
+        <div v-if="data.id?.length > 150" v-tooltip.bottom="{ value: data.id }">
+          {{ data.id }}
+        </div>
+        <div v-else>
+          {{ data.id }}
+        </div>
+      </template>
+    </Column>
+    <Column header="Updated date" :hidden="displayInfo ? true : false">
       <template #body="{ data }">
         {{ formatDateLong(data.updatedAt) }}
       </template>
     </Column>
+    <Column header="Actions" headerStyle="width: 15%" headerClass="header-right" bodyClass="content-right actions-buttons">
+      <template #body="{ data }">
```

Comment:
Any way to make this column always have a fixed size instead of relative? On relatively smaller viewports the buttons begin flowing around oddly.

Reply:
Similar to the fix in PR-8. Forced column to be 200px.